prisma-fmt-wasm/README.md

```diff
@@ -24,3 +24,25 @@ node -e "const prismaFmt = require('@prisma/prisma-fmt-wasm'); console.log(prism
 - The [nix build definition](https://github.com/prisma/prisma-engines/blob/main/prisma-fmt-wasm/default.nix)
     - It gives us a fully reproducible, thoroughly described build process and environment. The alternative would be a bash script with installs through `rustup`, `cargo install` and `apt`, with underspecified system dependencies and best-effort version pinning.
     - You can read more about nix on [nix.dev](https://nix.dev/) and the [official website](https://nixos.org/).
+
+## Local Dev with Language-Tools
+When implementing features for `language-tools` in `prisma-engines`, to sync with your local dev environment for the `language-server`, one can do the following:
+
+### On first setup
+Install the latest Rust version
+```
+cargo update -p wasm-bindgen
+cargo install -f wasm-bindgen-cli@version
+```
+- version is defined in `prisma-fmt-wasm/cargo.toml`
```

Comment:
```suggestion
### On first setup
```
# Install the latest Rust version with `rustup`
# or update the latest Rust version with `rustup`
rustup update
cargo update -p wasm-bindgen
# Check the version defined in `prisma-fmt-wasm/cargo.toml` for `wasm-bindgen` and replace `version` below:
cargo install -f wasm-bindgen-cli@version
```